website/docs/reference/rbac.md

```diff
@@ -79,7 +79,7 @@ You can assign the following root permissions:
 | Create frontend API tokens | Create frontend API tokens. |
 | Update frontend API tokens | Update frontend API tokens. |
 | Delete frontend API tokens | Delete frontend API tokens. |
-| Read client API tokens     | View [client API tokens](./api-tokens-and-client-keys#client-tokens).     |
+| Read client API tokens     | View [client API tokens](./api-tokens-and-client-keys#backend-tokens).     |
```

Comment:
Is the permission name not changing?

Reply:
Not now, because it's about the "Client API" which we're not changing. Maybe we're more specific saying "Client features API" but that'd ignore the metrics...

Reply:
Yes! I checked the code and this permission is actually specific to the backend. Maybe I should update it... added a db migration, so now we can update these

```suggestion
| Read Backend tokens     | View [backend API tokens](./api-tokens-and-client-keys#backend-tokens).     |
```